### PR TITLE
Fixes https://github.com/lightblue-platform/lightblue-rest/issues/208

### PR DESCRIPTION
**Pull request by dcrissman**
Fixes https://github.com/lightblue-platform/lightblue-rest/issues/208

Exposes IdentityManagement to client testing.

### CODE REVIEW

File: lightblue-client-integration-test/src/test/java/com/redhat/lightblue/client/integration/test/TestLightblueExternalResource.java [hunk 1]
```diff
@@ -3,35 +3,46 @@
...
+    public void before() throws IOException, InterruptedException {
+        lightblue.getControllerInstance().cleanupMongoCollections("country");
+
+        if (lightblue.getControllerInstance().getIdentityManager() != null) {
+            lightblue.changeIdentityManager(null);
+
+            //TODO remove sleep. There is some sort of timing issue with the server restart.
+            //DeploymentManager might be the key, but no way to get access to it today
+            // https://github.com/undertow-io/undertow/blob/master/examples/src/main/java/io/undertow/examples/servlet/ServletServer.java
+            Thread.sleep(5000);
```
Comment (paterczm) [inline lightblue-client-integration-test/src/test/java/com/redhat/lightblue/client/integration/test/TestLightblueExternalResource.java]: This sleep is only needed if you're changing identity manager, right? Maybe put it in changeIdentityManager?

File: lightblue-client-integration-test/src/test/java/com/redhat/lightblue/client/integration/test/TestLightblueExternalResource.java [hunk 2]
```diff
@@ -45,7 +56,26 @@ public void test() throws Exception {
...
+        Country c = new Country();
+        c.setName("Poland");
+        c.setIso2Code("PL");
+        c.setIso3Code("POL");
+
+        DataInsertRequest insert = new DataInsertRequest(Country.objectType, Country.objectVersion);
+        insert.create(c);
+        insert.returns(Projection.includeFieldRecursively("*"));
+
+        LightblueDataResponse insertResponse = lightblue.getLightblueClient("fakeuser", "fakepassword").data(insert);
```
Comment (paterczm) [inline lightblue-client-integration-test/src/test/java/com/redhat/lightblue/client/integration/test/TestLightblueExternalResource.java]: Would also test with wrong user/pass.

**Comment by dcrissman** [comment]
@paterczm - requested changes made.

File: lightblue-client-integration-test/src/test/java/com/redhat/lightblue/client/integration/test/TestLightblueExternalResource.java (hunk 1 above)
Comment (dcrissman) [inline lightblue-client-integration-test/src/test/java/com/redhat/lightblue/client/integration/test/TestLightblueExternalResource.java]: done

**Pull request merged**